ArduCopter/switches.cpp

```diff
@@ -550,6 +550,7 @@ void Copter::do_aux_switch_function(int8_t ch_function, uint8_t ch_flag)
             break;
 
         case AUXSW_PRECISION_LOITER:
+#if PRECISION_LANDING == ENABLED
```

Comment:
I think that it is better to include case statements.
Code size will also be reduced slightly.